src/components/dynamic/admin/dashboards/Toolbar.jsx

```diff
@@ -139,12 +139,17 @@ const Toolbar = ({
       <div className="flex w-1/3">
         <FaTrashAlt
           data-cy="delete"
-          onClick={() =>
-            setPopup({
-              ...popup,
-              visible: true,
-            })
-          }
+          onClick={() => {
+            const numSelectedObjects = objects.filter((a) => a.selected).length;
+            if (numSelectedObjects) {
+              setPopup({
+                ...popup,
+                visible: true,
+              });
+            } else {
+              toast("❌ Select row(s) before pressing the delete button");
```

Comment:
u can remove the else statement, the if statement should only be triggering toasts and returning from the function. check other functions that have a toast to see how they handle the error handling